code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -487,6 +488,22 @@
 	to_chat(src, SPAN_NOTICE("You [C.toggled ? "disable" : "enable"] [C.name]."))
 	C.toggled = !C.toggled
 
+/obj/item/robot_module/janitor/verb/toggle_mop()
+	set category = "Robot Commands"
+	set name = "Toggle Mop"
+	set desc = "Toggle the integrated mop."
+	if (!mopping)
+		mopping = 1
+		to_chat(usr, "You enable your integrated mopping system.")
+		src.visible_message("\ [usr]'s integrated mopping system rumbles to life.")
```

Comment:
```suggestion
		src.visible_message(SPAN_NOTICE("[usr]'s integrated mopping system rumbles to life."), SPAN_NOTICE("You enable your integrated mopping system."))
```
hmmmm.

`src` in this case is a module hidden away in the borg. This message probably doesn't play to anyone. Use `usr` as the source of the message, `usr` being the player that clicks the button.

Also, second arg for visible message in that case is what the user sees, first is what everyone else sees.

Reply:
And remove the to_chat in this case.